### DISCUSSION TITLE
Generate translated pictures flipped

### DISCUSSION THREAD

**Discussion by eslam650** [xulihang/ImageTrans-docs#592] [Q&A] [answered]
**Hello, Is there away to generate translated pictures flipped instead of using generate editable PSD files ?**

**Comment by xulihang** [comment]
What is it for? It is not easy to implement this directly in ImageTrans.

**Reply by eslam650** [reply]
to translate comics and make it readable from right to left.

**Comment by xulihang** [comment, marked as answer]
I just added a project setting for this in v2.9.0. You can download v2.9.0 to have a try.


![image](https://github.com/xulihang/ImageTrans-docs/assets/5462205/7ab46e0b-f327-4ad6-9f30-5dfbff319883)

**Reply by eslam650** [reply]
Great job. I will try it. thanks for your hard work.
